### PR TITLE
Migration to androidx

### PR DESCRIPTION
**Pull request by RingerJK**
Related: #55

### CODE REVIEW

File: gradle/versions.gradle [hunk 1]
```diff
@@ -8,7 +8,8 @@ ext {
     compile_sdk_version = 28
     build_tools_version = '28.0.3'
 
-    support_lib = '27.1.1'
+    android_x_annotation = '1.1.0-alpha01'
```
Comment (yunikkk) [inline gradle/versions.gradle]: Alpha release here, too.

Comment (RingerJK) [reply gradle/versions.gradle]: @yunikkk done

File: app/src/main/res/layout/activity_map.xml [hunk 2]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".activity.map.MapActivity">
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                                                   xmlns:app="http://schemas.android.com/apk/res-auto"
```
Comment (yunikkk) [inline app/src/main/res/layout/activity_map.xml]: Formatting changed

Comment (RingerJK) [reply app/src/main/res/layout/activity_map.xml]: done

File: gradle/versions.gradle [hunk 3]
```diff
@@ -8,7 +8,8 @@ ext {
     compile_sdk_version = 28
     build_tools_version = '28.0.3'
 
-    support_lib = '27.1.1'
+    android_x_annotation = '1.0.2'
```
Comment (yunikkk) [inline gradle/versions.gradle]: Besides is `android_x_annotation` used anywhere?

Comment (RingerJK) [reply gradle/versions.gradle]: yep

**Comment by RingerJK** [comment]
@yunikkk PTAL

**Pull request merged**